frontend/src/worksheets/CreateWorkSheetForm.js

```diff
@@ -45,10 +46,13 @@ function CreateWorkSheetForm({
               localDateTime: getCurrentDate(),
               workerSignature: '',
               proofOfEmployment: '',
-              status: '',
+              worksheetStatus: 'CREATED',
             }}
             validationSchema={schema}
             onSubmit={(values) => {
+              if (finalize) {
+                values.worksheetStatus = 'REPORTED'
+              }
```

Comment:
És ha rányomok, hogy lezárom, nem sikerül a kérés, majd inkább mégis a mentésre nyomok, akkor reported status-szal megy be? :) Itt kellene else ág szerintem.

Reply:
Arra gondolsz, ha a 'Mentés és készre jelentés'-re kattintok, de pl. nincs minden kitöltve, majd miután kitölttem, a mentésre megyek? Mert akkor created lesz. Mivel a mentés gombnál visszaállítom false-ra.

Reply:
Vagy server hibára gondolsz? Mert így jobban átgondolva, abban az esetben tényleg nem jó.